tests/framework/artifacts.py

```diff
@@ -178,6 +322,40 @@ def _fetch_artifacts(self,
                                                 artifact_type=artifact_type))
         return artifacts
 
+    def snapshots(self, keyword=None):
+        """Return snapshot artifacts for the current arch."""
+        # Snapshot artifacts are special since they need to contain
+        # a variable number of files: mem, state, disks, ssh key.
+        # To simplify the way we retrieve and store snapshot artifacts
+        # we are going to group all snapshot file in a folder and the
+        # "keyword" parameter will filter this folder name.
+        #
+        # Naming convention for files within the snapshot below.
+        # Snapshot folder /ci-artifacts/snapshots/x86_64/fc_snapshot_v0.22:
+        # - vm.mem
+        # - vm.vmstate
+        # - disk0 <---- this is the root disk
+        # - disk1
+        # - diskN
+        # - ssh_key
+
+        artifacts = []
+        prefix = ArtifactCollection.ARTIFACTS_ROOT
```

Comment:
Could this be initialized in one line?
```
prefix = ArtifactCollection.ARTIFACTS_ROOT + ArtifactCollection.ARTIFACTS_SNAPSHOTS
``` 
?

Reply:
That might be a very long line according to the linter.